zlp-scheduler/db/migrate/20200417011508_add_active_to_terms.rb

```diff
@@ -0,0 +1,5 @@
+class AddActiveToTerms < ActiveRecord::Migration[5.0]
+  def change
+    add_column :terms, :active, :boolean
```

Comment:
Apologies, we got our wires crossed. I've already added this column for you, you don't need this migration.